### PR TITLE
Remove marine_clean_aerosol from default AHI rayleigh_corrected modifier

### PR DESCRIPTION
**Pull request by djhoese**
This was brought up on slack a while ago by me, but was never acted on. I noticed that ABI only uses "rayleigh_only" for its pyspectral settings for the `rayleigh_corrected` modifier while the AHI version uses `marine_clean_aerosol`. When discussed with @adybbroe, he said:

```
It might "just" be a coincidence but from what I recall my own thoughts way back I thought the AHI disk is having more ocean (Pacific) views than ABI and that could motivate a LUT more adapted to marine type aerosols. But very unscientific - bottom line: we should keep the rayleigh_only as default for both until someone proves we should change!
```

And @simonrp84 said:

```
I agree with Adam that both should be Rayleigh only for now. It's the safer option and provides better consistency, using a marine aerosol over, say, the polluted regions of China or India will not be terribly good.
```

Even the new JMA true color reproduction made a new version of rayleigh_corrected with this change made.

Here's what I get for the two versions. Original `marine_clean_aerosol`:

image

New rayleigh_only:

image

If you swap between them you can see major differences on the outer parts of the disk and a general brightness/contrast difference.

Related: https://github.com/ssec/polar2grid/issues/403
 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [ ] Fully documented 
 - [ ] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

**Review by adybbroe — APPROVED** [review]
Looks good. I agree, we should stick to the rayleigh-only for the time being and be consistent across sensors

**Comment by adybbroe** [comment]
From the two images I can see the differences but I can't say one is better than the other. But that does not argue against using the Rayleigh-only option. Thanks @djhoese !

File: satpy/etc/composites/ahi.yaml [hunk 1]
```diff
@@ -2,24 +2,11 @@ sensor_name: visir/ahi
...
-    - solar_azimuth_angle
-    - solar_zenith_angle
-
-  no_aerosol_rayleigh_corrected:
     modifier: !!python/name:satpy.modifiers.PSPRayleighReflectance
     atmosphere: us-standard
     aerosol_type: rayleigh_only
     prerequisites:
-    - name: B03
+    - wavelength: B03
```
Comment (mraspaud) [inline satpy/etc/composites/ahi.yaml:9]: How can this work?

Comment (simonrp84) [inline satpy/etc/composites/ahi.yaml:9]: Why did you change this? B03 isn't a wavelength :-)
It should either remain as `name` or switch to `- wavelength: 0.64`

**Review by simonrp84 — CHANGES_REQUESTED** [review]
I've added one comment. Otherwise it looks OK.

@adybbroe Over asia the Rayleigh composite looks better to me, but near the Earth's limb the marine aerosol looks better. Guess that's as expected, and I think the bits near the ilmb could be fixed by toning down the sunz correction.

File: satpy/etc/composites/ahi.yaml (hunk 1 above)
Comment (djhoese) [reply satpy/etc/composites/ahi.yaml:9]: Maybe I generated the image and then made this change right before committing. You know, like a bad TDD developer.

Comment (djhoese) [reply satpy/etc/composites/ahi.yaml:9]: The two rayleigh-based modifiers that were here were essentially the same except for the aerosol type. In general I think we've been specifying bands explicitly in sensor-specific configs so I went with B03 here, but made a typo.

```suggestion
    - name: B03
```

**Comment by simonrp84** [comment]
LGTM!
